misc/peer-id-generator/Cargo.toml

```
@@ -11,5 +11,5 @@ categories = ["network-programming", "asynchronous"]
 publish = false
 
 [dependencies]
-libp2p-core = { version = "0.19.0", path = "../../core" }
+libp2p-core = { version = "0.20", path = "../../core" }
```

Actually, since this crate is not published, we may remove the version constraint:

```suggestion
libp2p-core = { path = "../../core" }
```